sdk/formrecognizer/azure-ai-formrecognizer/pom.xml

```diff
@@ -15,8 +15,8 @@
   <artifactId>azure-ai-formrecognizer</artifactId>
   <version>4.1.0-beta.3</version> <!-- {x-version-update;com.azure:azure-ai-formrecognizer;current} -->
 
-  <name>Microsoft Azure client library for Form Recognizer</name>
-  <description>This package contains the Microsoft Azure Cognitive Services Form Recognizer SDK.</description>
+  <name>Microsoft Azure client library for Document Intelligence</name>
+  <description>This package contains the Microsoft Azure Cognitive Services Document Intelligence SDK.</description>
```

Comment:
I believe the service is using the name "Azure AI Document Intelligence" now, without the "Cognitive Services" part.

Reply:
I think since the resource creation actually still lives under Cognitive services wanted to keep this until it is moved to be under AI.